src/xdebugger.cpp

```diff
@@ -128,8 +130,16 @@ namespace xpyt
 
     bool debugger::start_debugpy()
     {
-        // debugpy has to be started in the main thread
-        std::string code = "import debugpy\ndebugpy.listen((\'" + m_debugpy_host + "\'," + m_debugpy_port + "))";
+        // import debugpy 
+        std::string code = "import debugpy;";
+        // specify sys.executable
+        auto it = m_debugger_config.find("python");
+        if (it != m_debugger_config.end())
+        {
+            code += "debugpy.configure({\'python\': \'" + it->template get<std::string>()  + "\'});";
```

Comment:
I think you can remove the `;` before the last `"`

Reply:
Don't we do `code += "debugpy.listen` right after?

Reply:
Ha my bad, I was persuaded the statements were separated with ' \n'.